### PR TITLE
Hard code release version

### PR DESCRIPTION
**Pull request by leeyr338**
Hard code release version. 

## LLT test
- From RPC interface.
Screenshot from 2019-09-11 12-33-05

- From micro-service binary:
Screenshot from 2019-09-11 12-36-06

### CODE REVIEW

File: cita-auth/build.rs [hunk 1]
```diff
@@ -18,7 +18,9 @@ use std::env;
 
 use util::build_info::gen_build_info;
 
+const VERSION: &str = "1.0.0";
```
Comment (kaikai1024) [inline cita-auth/build.rs]: What about put this at a public common type. then other modules use it directly?
```
pub const VERSION
```

Comment (leeyr338) [reply cita-auth/build.rs]: Yeah, that's a good option!
But I want to keep a separate version of the evolution capability for each service.

Comment (kaikai1024) [reply cita-auth/build.rs]: > But I want to keep a separate version of the evolution capability for each service.

For this situation, just do not use the common version value and define own variable.
What i said can avoid that we miss some variables that shoud be changed to new version.

It's ok for me.  maybe discuss it later.

Comment (leeyr338) [reply cita-auth/build.rs]: Okay!
As you say, avoiding miss some variables be changed is a thinking aspect.
I have thought to define the `pub const VERSION` in cita-common, but I don't think this is a good option.

**Pull request merged**